File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function (nums) {
    let range = nums.length;

    let actVal = (range * (range + 1)) / 2;

    let calVal = 0;

    for (let i = 0; i < nums.length; i++) {
        calVal += nums[i];
    }

    let ans = actVal - calVal;

    return ans;
};

File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int numsSize) {
    int actVal = (numsSize * (numsSize + 1)) / 2;

    int calVal = 0;

    for(int i = 0; i < numsSize; i++) {
        calVal += nums[i];
    }

    int ans = actVal - calVal;

    return ans;
}